spec/integration/type/user_spec.rb

```diff
@@ -15,7 +15,7 @@
   include PuppetSpec::Files
   include PuppetSpec::Compiler
 
-  context "when set to purge ssh keys from a file" do
+  context "when set to purge ssh keys from a file", :if => Puppet::Type.type(:ssh_authorized_key) do
```

Comment:
Should these tests be moved to the module?

Reply:
Yes, they should. That came up in chat, but hasn't been done yet.